src/definition/externalComponent/IWebHooks.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * This interface provides webhooks to the game. You can set
+ * the target URL for each webhook event according to your demand.
+ */
+export interface IWebHooks {
```

Comment:
I'm not so sure about this approach, though.

I was thinking about making this `sessionEnd` an event the app could subscribe to, similar to how [PostMessageSent event works](https://github.com/RocketChat/Rocket.Chat.Apps-engine/blob/master/src/definition/messages/IPostMessageSent.ts#L5)

So an app class would implement an `IExternalComponentSessionEnded` interface, implement an `executeExternalComponentSessionEnded` method that receives the important data from the engine, and then would be able to do whatever it wants with this.

In the Rocket.Chat side then, you would need to trigger this event with [something like this](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/lib/server/functions/sendMessage.js#L211)

Does it make sense?

Reply:
Yes, I will try the approach you suggested.